pkg/usagestats/seed_test.go

```diff
@@ -70,6 +71,8 @@ func Test_Memberlist(t *testing.T) {
 	// create a first memberlist to get a valid listening port.
 	initMKV := createMemberlist(t, 0, -1)
 
+	var wg sync.WaitGroup
+	wg.Add(10)
```

Comment:
Since the `10` constant here is mentioned twice, perhaps this add should be inside the `for` loop and then just add `1` per iteration.